test/nodetool/conftest.py

```diff
@@ -141,6 +141,12 @@ def scylla_only(request):
         pytest.skip('Scylla-only test skipped')
 
 
+@pytest.fixture(scope="function")
+def cassandra_only(request):
+    if request.config.getoption("nodetool") != "cassandra":
+        pytest.skip('Cassandra-only test skipped')
```

Comment:
Ideally, such a fixture should not exist:
1. If this is a cassandra feature that **should** be in Scylla, it's a Scylla bug, and it should be marked "@xfail", not "cassandra_only".
2. If this is a cassandra feature that should never be a Scylla, we shouldn't have a test for it - we're not writing a test suite for Cassandra.

Reply:
Ok, I understand better now why you added cassandra_only - because you wanted the Java ("cassandra") nodetool to behave differently than the C++ one. More on this below.

Reply:
yes, it's not a cassandra-only feature, but rather cassandra-specific implementation detail that is tested (in a grey-box way)